convex/testingFunctions.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    customAction,
+    customMutation,
+    customQuery,
+  } from "convex-helpers/server/customFunctions";
+  import { action, mutation, query } from "./_generated/server";
+import schema from "./schema";
+
+// Wrappers to use for function that should only be called from tests
+export const testingQuery = customQuery(query, {
+  args: {},
+  input: async (_ctx, _args) => {
+    if (process.env.IS_TEST === undefined) {
+      throw new Error(
+        "Calling a test only function in an unexpected environment"
+      );
+    }
+    return { ctx: {}, args: {} };
+  },
+});
```

Comment:
FWIW it'll vary from developer to developer whether they're using `customCtx` or `verify` so I kind of like the most explicit thing so there's no potential confusion around what to return.

Like if I was already using a `customQuery` handling a sessionId argument, I think it would be more obvious how to add this `IS_TEST` check to make a test only version of my query wrapper than if I used `customCtx` / `verify`